tests/openssl_moretest/host/digest_tests.cpp

```diff
@@ -0,0 +1,349 @@
+     0,
+     {_Neg, _Neg, _Neg, _Neg, _Neg},
+     {0, 0, 0, 0, 0}},
+
+    {"SSLeay_version",
+     e_SSLeay_version,
+     2,
+     {sizeof(int), _ELLEPH, _Neg, _Neg, _Neg},
+     {S_FIXLEN_IO | S_RAND, S_FIXLEN_O, 0, 0, 0}},
+
+    {"EVP_EncodeInit",
+     e_EVP_EncodeInit,
+     1,
+     {sizeof(EVP_ENCODE_CTX), _Neg, _Neg, _Neg, _Neg, _Neg, _Neg, _Neg, _Neg},
+     {S_FIXLEN_IO | S_RAND, 0, 0, 0, 0, 0, 0, 0, 0}},
+    {"EVP_EncodeUpdate",
+     e_EVP_EncodeUpdate,
+     5,
+     {sizeof(EVP_ENCODE_CTX),
+      _Neg,
+      sizeof(size_t),
+      _Neg,
+      sizeof(size_t),
+      _Neg,
+      _Neg,
+      _Neg,
+      _Neg},
+     {S_FIXLEN_IO | S_RAND,
+      S_VARLEN_I | S_RAND,
+      S_LEN,
+      S_VARLEN_I | S_RAND,
+      S_FIXLEN_IO,
+      0,
+      0,
+      0,
+      0}},
+    {"EVP_DecodeInit",
+     e_EVP_DecodeInit,
+     1,
+     {sizeof(EVP_ENCODE_CTX), _Neg, _Neg, _Neg, _Neg, _Neg, _Neg, _Neg, _Neg},
+     {S_FIXLEN_IO | S_RAND, 0, 0, 0, 0, 0, 0, 0, 0}},
+    {"EVP_DecodeUpdate",
+     e_EVP_DecodeUpdate,
+     5,
+     {sizeof(EVP_ENCODE_CTX),
+      _Neg,
+      sizeof(size_t),
+      _Neg,
+      sizeof(size_t),
+      _Neg,
+      _Neg,
+      _Neg,
+      _Neg},
+     {S_FIXLEN_IO | S_RAND,
+      S_VARLEN_I | S_RAND,
+      S_LEN,
+      S_VARLEN_I | S_RAND,
+      S_FIXLEN_IO,
+      0,
+      0,
+      0,
+      0}},
+    {"EVP_DecodeFinal",
+     e_EVP_DecodeFinal,
+     5,
+     {sizeof(EVP_ENCODE_CTX),
+      _Neg,
+      sizeof(size_t),
+      _Neg,
+      sizeof(size_t),
+      _Neg,
+      _Neg,
+      _Neg,
+      _Neg},
+     {S_FIXLEN_IO | S_RAND,
+      S_VARLEN_I | S_RAND,
+      S_LEN,
+      S_VARLEN_I | S_RAND,
+      S_FIXLEN_IO,
+      0,
+      0,
+      0,
+      0}},
+
+    {"EVP_DigestInit_ex",
+     e_EVP_DigestInit_ex,
+     1,
+     {sizeof(EVP_MD_CTX), _Neg, _Neg, _Neg, _Neg},
+     {S_FIXLEN_IO | S_RAND | S_CMP_EVP_MD_CTX, 0, 0, 0, 0}},
+    {"EVP_DigestUpdate",
+     e_EVP_DigestUpdate,
+     3,
+     {sizeof(EVP_MD_CTX), _Neg, sizeof(size_t), _Neg, _Neg},
+     {S_FIXLEN_IO | S_RAND | S_CMP_EVP_MD_CTX,
+      S_VARLEN_I | S_RAND,
+      S_LEN,
+      0,
+      0}},
+    {"EVP_DigestFinal_ex",
+     e_EVP_DigestFinal_ex,
+     3,
+     {sizeof(EVP_MD_CTX), EVP_MAX_MD_SIZE, sizeof(unsigned int), _Neg, _Neg},
+     {S_FIXLEN_IO | S_RAND | S_CMP_EVP_MD_CTX,
+      S_FIXLEN_IO | S_RAND,
+      S_FIXLEN_IO | S_RAND,
+      0,
+      0}},
+
+};
+
+t_openssl_schema* get_digest_schema()
+{
+    return _openssl_digest;
+}
+
+uint get_digest_schema_length()
+{
+    return sizeof(_openssl_digest) / sizeof(_openssl_digest[0]);
+}
+
+class CSchemaChecker_Digest : public CSchemaChecker
+{
+  public:
+    CSchemaChecker_Digest(t_openssl_schema* schema, uint schema_size)
+        : CSchemaChecker(schema, schema_size)
+    {
+    }
+};
+
+oe_result_t schema_test_digest(oe_enclave_t* enclave)
+{
+    oe_result_t result;
+    int retval;
+    uint num_apis = get_digest_schema_length();
```

Comment:
```suggestion
    uint api_count = get_digest_schema_length();
```
https://github.com/openenclave/openenclave/blob/master/docs/Community/DevelopmentGuide.md says:

> DON'T use abbreviations unless they are already well-known terms known by users (e.g., "app", "info"), or are already required for use by developers (e.g., "min", "max", "args"). Examples of bad use would be num_widgets instead of widget_count, and opt_widgets instead of option_widgets or optional_widgets.